src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+          "startKey": "8okSYW1zdGVyZGFtAAE=",
+          "endKey": "8okSYW1zdGVyZGFtAAESszMzMzMzQAD/gAD/AP8A/wD/AP8A/yMAAQ==",
+          "internalReplicas": [
+            {
+              "nodeId": 8,
+              "storeId": 8,
+              "replicaId": 5,
+              "type": 0
+            },
+            {
+              "nodeId": 4,
+              "storeId": 4,
+              "replicaId": 7,
+              "type": 0
+            },
+            {
+              "nodeId": 3,
+              "storeId": 3,
+              "replicaId": 6,
+              "type": 0
+            }
+          ],
+          "nextReplicaId": 8,
+          "generation": "40",
+          "stickyBit": {
+            "wallTime": "0",
+            "logical": 0,
+            "synthetic": false
+          }
+        },
+        "config": {
+          "rangeMinBytes": "134217728",
+          "rangeMaxBytes": "536870912",
+          "gcPolicy": {
+            "ttlSeconds": 14400,
+            "protectionPolicies": [
+            ],
+            "ignoreStrictEnforcement": false
+          },
+          "globalReads": false,
+          "numReplicas": 3,
+          "numVoters": 0,
+          "constraints": [
+            {
+              "numReplicas": 0,
+              "constraints": [
+                {
+                  "type": 1,
+                  "key": "region",
+                  "value": "europe-west1"
+                }
+              ]
+            }
+          ],
+          "voterConstraints": [
+          ],
+          "leasePreferences": [
+          ],
+          "rangefeedEnabled": false,
+          "excludeDataFromBackup": false
+        }
+      },
+      ...
+    ],
+    "unavailable": [
+    ],
+    "unavailableNodeIds": [
+    ]
+  }
+}
+~~~
+
+##### Replication status - under-replicated ranges
+
+The following example assumes you are running a newly started, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+By default, this geo-distributed demo cluster will not have any [under-replicated ranges](ui-replication-dashboard.html#under-replicated-ranges).
+
+To put the cluster into a state where some [ranges](architecture/overview.html#architecture-range) are under-replicated, issue the following [`ALTER TABLE ... CONFIGURE ZONE`](alter-table.html#configure-zone) statement, which tells it to store 9 copies of each range underlying the `rides` table.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE rides CONFIGURE ZONE USING num_replicas=9;
+~~~
+
+Once the statement above executes, the cluster will rebalance so that it's storing 9 copies of each range underlying the `rides` table. During the time it takes for the rebalancing to occur, these ranges will be considered under-replicated, since there are not yet as many copies (9) of each range as you have just specified.
```

Comment:
Suggesting this word change now that you've confirmed I correctly understood this...

```suggestion
Once the statement above executes, the cluster will up-replicate so that it's storing 9 copies of each range underlying the `rides` table. During the time it takes for the rebalancing to occur, these ranges will be considered under-replicated, since there are not yet as many copies (9) of each range as you have just specified.
```